docs/docs/user_guide/sidebar/common/studies/studies_page.md

```diff
@@ -0,0 +1,30 @@
+---
+id: studies_page
+title: Studies Page
+sidebar_label: Studies Page
+---
+
+The Studies page provides a central location for creating Studies, uploading files, managing Study permissions, and creating new Workspaces.
+
+### Changing Organization Study Permissions
+To change permissions for an Organization Study:
+
+1.	Click the **Permissions** button for the Study
+2.	Click the pencil icon beside the Users level
+3.	Update permissions as needed.
+
+:::note
+Changing a user’s permissions for an External Study will result in all existing Workspaces for that user losing all access to the Study. This will be addressed in a future release.
+:::
+
+### Creating a Study and Uploading Files
+To create a Study:
+
+1.	Click **Create Study**.
+2.	Provide a unique ID, type, Name, Description, and Project ID.
+3.	Click **Create Study**.
+
+To upload files:
+
+4.	Click **Upload Files**.
```

Comment:
Probably just call out here that this feature isn't available for external studies